sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/ColumnarBatch.java

```diff
@@ -232,6 +234,96 @@ public MapData getMap(int ordinal) {
     public Object get(int ordinal, DataType dataType) {
       throw new NotImplementedException();
     }
+
+    @Override
+    public void update(int ordinal, Object value) {
```

Comment:
yes, this shouldn't be on the hot path. It was just a method that needed to be overridden while implementing `MutableRow`.